components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,60 +10,145 @@ toc = true
-We recommend deleting these legacy policies and setting up new IAM v2 policies in their place.
-
-In the browser: Navigate to the *Policies* page under the **Settings**  tab.
+For those API methods that take JSON data (typically `create` and `update` methods)
+you can provide that data to the REST endpoint in a variety of ways.
+If it is a relatively small size payload, you can include it in a `curl` command inline, e.g.
 
-### Getting a Policy
+```bash
+curl -sH "api-token: $TOKEN" -d '<your JSON here>' ...
+```
 
-In order to fetch a policy on the command-line, you must know its ID.
-This may be obtained by first fetching the list of all policies; see [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}).
-Find the `id` field for the policy of interest--for example, the ID from the default administrator policy is `administrator-access`.
-Plug that in to the URL as shown:
+If the payload is larger, it is often convenient to store it in a file, then pass that file, e.g.,
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/administrator-access?pretty
+curl -sH "api-token: $TOKEN" -d @policy.json ...
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings**  tab, and then select an individual policy to open its details.
+## Notes on the Projects Property
 
-### Creating a Policy
+Teams, tokens, roles, and policies all contain a top-level `projects` property; you will see that in the example JSON for each resource in the following sections.
+This property associates the particular IAM resource with one or more projects.
+Users must have permissions for those projects to be able to view or modify those resources.
+This `projects` property may be empty, which would give access only to users included on policies specifying the special `(unassigned)` project designation.
+This `projects` property may **not** specify `*` to indicate all projects.
 
-Create a policy by composing JSON with all the necessary policy data.
+To create those permissions for users, there is a separate and distinct `projects` property on each policy statement.
+This `projects` property may **not** be empty; you must either specify specific projects or use a wildcard (`*`) to indicate all projects.
 
-This example policy is made up of two statements, each enforcing a different set of permissions.
-The first policy statement grants users list view access.
-The second statement grants editor access via the `editor` role.
-You can see the actions contained in the `editor` role in the [Roles section]({{< relref "iam-v2-api-reference.md#roles" >}}).
+Here is a summary:
+
+Type               | May be empty ? | Include * ? | May specify (unassigned) ?
+-------------------|----------------|-------------|------------
+top-level projects |    yes         |   no        |  no
+statement projects |    no          |   yes       |  yes
```

Comment:
Sure feels like we should have two different terms instead of just "projects". 🤷‍♂